gme/Music_Emu.cpp

```diff
@@ -241,10 +248,22 @@ blargg_err_t Music_Emu::seek( long msec )
 	return seek_samples( msec_to_samples( msec ) );
 }
 
+blargg_err_t Music_Emu::seek_scaled(long msec)
+{
+	require(tempo_ > 0);
+	blargg_long frames = (msec / 1000.0) * sample_rate();
```

Comment:
Can you replace `blargg_long` with `int32_t`? It's planned replacement of all blaarg_long and blaarg_ulong with int32_t and uint32_t because in modern times (since C++98) presense of these has no sense.